test/plugin_functional/test_suites/core_plugins/ui_plugins.ts

```diff
@@ -97,5 +98,45 @@ export default function({ getService, getPageObjects }: PluginFunctionalProvider
         ).to.be('/core_plugin_b/system_request says: "System request? false"');
       });
     });
+
+    describe('Static assets', function() {
+      describe('Platform assets', function() {
+        it('exposes static assets', async () => {
+          await supertest.get('/ui/favicons/favicon.ico').expect(200);
+        });
+
+        it('returns 404 if not found', async function() {
+          await supertest.get('/ui/favicons/not-a-favicon.ico').expect(404);
```

Comment:
NIT: I'm getting lost with the `src/test` subfolders... Are the platform-exposed assets tests supposed to be in the `plugin_functional` or in the `functional` suite? These could even be in the `api_integration` suite as they don't requires an headless browser to run, even if not sure that can be considered as api integration testing?

Reply:
I put it here to be closer to the plugin tests. I will move platform tests in `core_app/integration_tests` folder.